woocommerce/woocommerce
# Issue 21515

### All Submissions:






### Changes proposed in this Pull Request:

Remove the erasure tools links from the descriptions of the erasure settings for users without the manage_privacy_options capability. In a typical WC install, this would apply to shop managers who have access to the settings screen.

Closes #21515 .

### How to test the changes in this Pull Request:

1. Log in as an administrator.
2. Go to the Accounts & Privacy settings tab.
3. Erasure links on Privacy settings should be present & accessible.
4. Repeat 1-3 as a shop manager. The Erasure links should not be in the setting descriptions.

### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Remove the erasure tools links from the descriptions of the erasure settings for users without the manage_privacy_options capability.
